tools/mkseg5defs.py

```diff
@@ -0,0 +1,60 @@
+import os
+import sys
+
+# This script converts MK64 course_texture struct tables to segment 5 address defines.
+# Segment 5 addresses are generated by the game using a texture course table.
+# This uses data size (uncompressed size) value on the same table to generate the address.
+
```

Comment:
Please add how to use the script

`# python3 mkseg5defs.py <offsets_input_file> <header_output_file>`

Reply:
I was documenting how the tool works and i forgot to add a "help" argument lol